web/packages/teleterm/src/ui/ConnectMyComputer/DocumentSetup/DocumentSetup.tsx

```diff
@@ -25,9 +25,7 @@ interface DocumentConnectMyComputerSetupProps {
   doc: types.DocumentConnectMyComputerSetup;
 }
 
-export function DocumentConnectMyComputerSetup(
-  props: DocumentConnectMyComputerSetupProps
-) {
+export function DocumentSetup(props: DocumentConnectMyComputerSetupProps) {
```

Comment:
Thoughts on keeping it long vs short?

Given that we're going to be left with things like `DocumentConnectMyComputerSetup` in document types, I'm not sure if it's worth it to refer to it in the short form in one place and in the long form in another.

I keep thinking of Facebook where they just keep all components on the same level, forcing them to come up with good unique names.

I think of documents as one of the primary building blocks of Connect, perhaps they should have clear identifiers. I don't feel super strong about it but something just tells me that it's the better option out of those two.

Reply:
Ah, that part was problematic for me, I started from `ui/ConnectMyComputer/DocumentConnectMyComputerSetup` and I thought that this long path looks so complicated. I decided use a short name in the directory and re-export the long one (but TBH I'm not a huge fan of `export as`).

I don't have a strong opinion on that too, I think we can come back to the long name so that our identifiers will stay consistent. 

Now the question is: should we do the same for `NavigationMenu` too? 🤔 Theoretically, it is not that important as the document.

Reply:
> Now the question is: should we do the same for `NavigationMenu` too? 🤔 Theoretically, it is not that important as the document.

I think it's important only for the documents. The IDs of documents will permeate the codebase as well as disk state, but `NavigationMenu` will be rendered in pretty much a single place.